### PR TITLE
Installation : content loader configuration

### PR DESCRIPTION
**Pull request by SebSept**
#### Description

Missing configuration needed to finish mdx integration.

The theme I use did use mdx by default, so I had to do this change.
It worth mentioning this point, I guess.

### CODE REVIEW

**Review by sarah11918 — COMMENTED** [review]
Hi @SebSept ! Thank you for contributing to the docs! I'm glad you figured out how to update your own theme, and appreciate you sharing that.

The code wasn't quite correct however, so I've updated it for our new content collections. And, I don't think I would put this in the section for *configuring the integration* since this is instead about *using* MDX in your Astro project. 

See what you think about my proposed changes! If you saw this in docs, would this have helped you with your project? Do you think this is helpful to readers?

**Comment by SebSept** [comment]
Thanks for the corrections ! Important point is the import statements missing.

In fact, this not my own theme but _astropaper_, found on the site : https://astro.build/themes/details/astropaper/ 

> If you saw this in docs, would this have helped you with your project? Do you think this is helpful to readers?
I double checked the documentation, tried some changes in the contents, rerun yarn installation, rerun the integration process, etc. I also wonder if it was related to the new version. Finally,  It took me a long time to figure that the _problem_ was in a file not mentioned in the documentation.  I'm new to Astro, so this is normal.
This is just my personal case, but for sure, having this information could have helped me a lot.
Maybe it's not relevant for the rest of us, maybe most themes include mdx files by default, I don't know.

**Comment by sarah11918** [comment]
Well, we do want docs to be helpful! And I don't think having this extra little section hurts anyone. If you say you would have found it a helpful reminder, then I think a section like you proposed is a good reminder that you CAN for sure use MDX in your content collections, even if you had to install MDX yourself.

So let's put this in docs and maybe it will help more people! 🙌

File: src/content/docs/en/guides/integrations-guide/mdx.mdx [hunk 1]
```diff
@@ -96,6 +96,25 @@ It also adds extra features to standard MDX, including support for Markdown-styl
 
 `.mdx` files must be written in [MDX syntax](https://mdxjs.com/docs/what-is-mdx/#mdx-syntax) rather than Astro’s HTML-like syntax.
 
+### Using MDX wth content collections
+
+To include MDX files in a content collection, make sure that your [collection loader](/en/guides/content-collections/#defining-the-collection-loader) is configured to load content from `.mdx` files:
+
+```js title="src/content.config.ts" ins="{mdx}"
```
Comment (sarah11918) [inline src/content/docs/en/guides/integrations-guide/mdx.mdx:103]:
```suggestion
```js title="src/content.config.ts" ins="mdx"
```

**Review by sarah11918 — APPROVED** [review]

**Pull request merged**